Example/SmileID/RootView.swift

```diff
@@ -11,6 +12,14 @@ struct RootView: View {
     init() {
         UITabBar.appearance().barTintColor = UIColor(SmileID.theme.backgroundLight)
         UITabBar.appearance().tintColor = UIColor(SmileID.theme.accent)
+        if let dsn = Bundle.main.object(forInfoDictionaryKey: "SentryDSN") as? String {
+            SentrySDK.start { options in
+                options.dsn = dsn
+                options.debug = true
+                options.tracesSampleRate = 1.0
+                options.profilesSampleRate = 1.0
+            }
```

Comment:
Can we also set the user id to the partner id, if partner id has already been set?